docs/connect-to-network.md

```diff
@@ -21,3 +21,15 @@ ledger_client = LedgerClient(cfg)
 ```
 
 A full list of chain identifiers, denominations and end-points can be found at the Cosmos [chain registry](https://github.com/cosmos/chain-registry/).
+
+
+
+### Custom prefix network:
+In case custom prefix network you need to create wallets with proper prefix specified:
```

Comment:
```suggestion

### Custom prefix network:
In case you are using network other than fetch.ai's, you can provide the custom prefix when creating the wallet:
```